README.md

```diff
@@ -226,7 +226,7 @@ XCUITest driver supports the following element attributes:
 
 Name | Description | Example
 --- | --- | ---
-name | Could contain either element's [identifier](https://developer.apple.com/documentation/xctest/xcuielementattributes/1500981-identifier?language=objc) or its [label](https://developer.apple.com/documentation/xctest/xcuielementattributes/1500692-label?language=objc), depending on which one is available first. Could also be `null` | 'hello'
+name | Could contain either element's [identifier](https://developer.apple.com/documentation/xctest/xcuielementattributes/1500981-identifier?language=objc) or its [label](https://developer.apple.com/documentation/xctest/xcuielementattributes/1500692-label?language=objc),  depending on which one is available first. Could also be `null`. The [`identifier` is UI Automation purpose](https://developer.apple.com/documentation/uikit/uiaccessibilityidentification/1623132-accessibilityidentifier), meanwhile the `label` is used for accessibility features in iOS, so we recommend you to use the identifity first for automation. | 'hello'
```

Comment:
Sorry, I don't quite understand what you are trying to say :(
I would be happy to help describing it

Reply:
This is kind of accessibilityIdentifier vs accessibilityLabel for `name`/`wdName`.

Our current logic returns `.identifier` first, and `.label` as the next. The `.identifier` is accessibilityIdentifier for UIAutomation. It is not used for accessibility stuff like VoiceOver while `.label` can be used by them.

So, ideally, the `.label` should not be used only for automation.
for example, a developer implements an element with multiple languages (e.g. English and Japanese).
```swift
struct ContentView: View {
    var body: some View {
        Text("Hello, world!")  // can be English or Japanese
            .padding()
    }
}
```

If no accessibilityLabel/accessibilityIdentifier exists in the element, `.label` will be "Hello, world!" (in English env case).
If a friend adds accessibilityLabel for automation like below,

```swift
struct ContentView: View {
    var body: some View {
        Text("Hello, world!")  // can be English or Japanese
            .padding().accessibilityLabel("for automation id")
    }
}
```

the `.label` will be `for automation id`. Then, voice over will read the `for automation id`.

```swift
struct ContentView: View {
    var body: some View {
        Text("Hello, world!")  // can be English or Japanese
            .padding().accessibilityIdentifier("for automation id")
    }
}
```

this will not affect to `.label` so voice over will keep 'Hello world!'.

Thus, wanted to address our recommendation for this part was `accessibilityIdentifier` for automation as a hint.
(Probably we haven't addressed like this tips in our documentation, but some web articles addresses `accessibilityLabel` for our accessibility_id, name)

Reply:
Thanks, it is now much cleaner to me. How about the following text:

It is recommended to prefer usage of [accessibilityIdentifier](https://developer.apple.com/documentation/uikit/uiaccessibilityidentification/1623132-accessibilityidentifier?language=objc) over [accessibilityLabel](https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel?language=objc) for automation purposes, since the identifier property is supposed to stay constant under different locales and does not affect VoiceOver scope.

Reply:
Seems great. Thanks!